uk.jnthn.backgammony.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "uk.jnthn.backgammony",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.dmd"
+    ],
+    "command": "backgammony",
+    "modules": [
+        {
+            "name": "gnubg",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./configure",
+                "make datadir=/app/share",
+                "install -D gnubg /app/bin/gnubg",
+                "install -D gnubg.wd /app/share/gnubg/gnubg.wd",
+                "install -D gnubg.weights /app/share/gnubg/gnubg.weights"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "ftp://ftp.gnu.org/gnu/gnubg/gnubg-release-1.06.002-sources.tar.gz",
+                    "sha256": "ce1b0b0c1900717cc598032a14cf8c0ee60faf24d84368b39922c0102983bc87"
+                }
+            ]
+        },
+        {
+            "name": "backgammony",
+            "buildsystem": "simple",
+            "build-options": {
+                "append-path": "/usr/lib/sdk/dmd/bin",
+                "build-args": ["--share=network"]
```

Comment:
Network access during build is not allowed to ensure builds are reproducible. This is also why your build failed. There's [a dub generator](https://github.com/flatpak/flatpak-builder-tools/tree/master/dub) which can generate snippets for you.